### PR TITLE
Changes version to 0.2.0

### PR DESCRIPTION
**Pull request by RassK**
## What
Changes version from 0.0.1 to 0.2.0

### CODE REVIEW

File: tracer/build/_build/Honeypot/IntegrationGroups.cs [hunk 1]
```diff
@@ -110,7 +110,7 @@ private async Task PopulatePackages()
                 {
                     IntegrationName = Name,
                     NugetPackageSearchName = packageName,
-                    MinVersion = "0.0.1",
+                    MinVersion = "0.2.0",
```
Comment (RassK) [inline tracer/build/_build/Honeypot/IntegrationGroups.cs:113]: this seems a mistage.

**Comment by pellared** [comment]
Can you also update `RELEASING.md` and  `CHANGELOG.md`?

File: tracer/test/snapshots/AspNetCoreIisMinimalApisTests.InProcess.NoFF.__path=__statusCode=200.verified.txt [hunk 2]
```diff
@@ -18,7 +18,7 @@
       net.peer.ip: ::1,
       runtime-id: Guid_1,
       signalfx.tracing.library: dotnet-tracing,
-      signalfx.tracing.version: 0.0.1.0,
+      signalfx.tracing.version: 0.2.0,
```
Comment (Kielek) [inline tracer/test/snapshots/AspNetCoreIisMinimalApisTests.InProcess.NoFF.__path=__statusCode=200.verified.txt:21]: Consider to revert all dump files and extend `Datadog.Trace.ClrProfiler.IntegrationTests.VerifyHelper.GetSpanVerifierSettings`

Comment (pellared) [reply tracer/test/snapshots/AspNetCoreIisMinimalApisTests.InProcess.NoFF.__path=__statusCode=200.verified.txt:21]: @mateuszrzeszutek  or @lachmatt  ? 😋 

> maybe verifying just signalfx.tracing.version: x.y.z, ? Simplifies releasing and verifying exact version is not that important.

Totally agree 👍

Comment (RassK) [reply tracer/test/snapshots/AspNetCoreIisMinimalApisTests.InProcess.NoFF.__path=__statusCode=200.verified.txt:21]: oops, wrong tag 😆

**Review by Kielek — APPROVED** [review]

**Review by pellared — APPROVED** [review]

**Pull request merged**